content/english/chatbot/activity-2.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Activity 2: Setup PandoraBots"
+draft: false
+weight: 5
+---
+
+## AIML
+We will be using AIML to create our chatbot. AIML stands for **Artificial Intelligence Markup Language**. If you have ever used HTML to make a website, you will find that AIML looks very similar. Before we can start writing AIML, we need to sign up for an account on PandoraBots.
+
+## Pandorabots
+Go to [https://www.pandorabots.com/](https://www.pandorabots.com/) and click on **Sign Up**. Enter an email and password to create an account. Alternatevely, you can sign up using an account from Google, Facebook, Twitter, or Github.

```

Comment:
We might want to state as a prerequisite that an email address is required to complete this workshop

Reply:
And maybe some more detailed sign up instructions.